clients/drcachesim/analyzer_multi.cpp

```diff
@@ -472,8 +472,53 @@ analyzer_multi_tmpl_t<RecordType, ReaderType>::analyzer_multi_tmpl_t()
         return;
     }
 
+    bool offline = !op_indir.get_value().empty() || !op_infile.get_value().empty();
+    // TODO i#7040: Add core-sharded support for online tools.
+    if (offline && !op_core_sharded.specified() && !op_core_serial.specified() &&
```

Comment:
Readability: Use named boolean vars to describe what the condition describes. Would be useful for such non-trivial conditions, even if the boolean var is single-use.

Reply:
Made a sharding_specified var.